go.mod

```diff
@@ -133,3 +133,7 @@ require (
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
 )
+
+replace (
+	github.com/cloudquery/cq-provider-sdk => ../cq-provider-sdk
```

Comment:
go.mod testing mode leak

Reply:
I think we can upgrade to latest SDK @amanenk